packages/datadog-instrumentations/src/mongodb.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+require('./mongodb-core')
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+// collection methods with filter
+const collectionMethodsWithFilter = [
+  'count',
+  'countDocuments',
+  'deleteMany',
+  'deleteOne',
+  'find',
+  'findOneAndDelete',
+  'findOneAndReplace',
+  'replaceOne'
+] // findOne is ignored because it calls to find
+
+const collectionMethodsWithTwoFilters = [
+  'findOneAndUpdate',
+  'updateMany',
+  'updateOne'
+]
+
+const startCh = channel('datadog:mongodb:collection:filter:start')
+
+addHook({ name: 'mongodb', versions: ['>=3.3'] }, mongodb => {
```

Comment:
I have to include the `file` option in order to make it work in my debugging app

```suggestion
addHook({ name: 'mongodb', file: 'lib/collection.js', versions: ['>=3.3'] }, mongodb => {
```

Reply:
false alarm